src/stirling/source_connectors/socket_tracer/socket_trace_connector.cc

```diff
@@ -90,7 +90,7 @@ DEFINE_bool(stirling_enable_nats_tracing, true,
 DEFINE_bool(stirling_enable_kafka_tracing, true,
             "If true, stirling will trace and process Kafka messages.");
 DEFINE_bool(stirling_enable_mux_tracing,
-            gflags::BoolFromEnv("PL_STIRLING_TRACER_ENABLE_MUX", false),
+            gflags::BoolFromEnv("PL_STIRLING_TRACER_ENABLE_MUX", true),
```

Comment:
We'll want to leave this as false until we can solve the BPF instruction count limit.